server/views/pages/event.njk

```diff
@@ -1,8 +1,14 @@
 {% extends 'layout/default.njk' %}
 
 {% block pageMeta %}
-  {% component 'open-graph', { pageConfig: pageConfig, article: event } %}
-  {% component 'twitter-card', { pageConfig: pageConfig, article: event } %}
+  {% set metaContent = {
+    type: 'wellcome:event' + '.' + event.format,
```

Comment:
What format does the namespace have to be in then, as it looks like this needs to be specified?